pkg/v1/providers/infrastructure-vsphere/v1.3.1/ytt/overlay.yaml

```diff
@@ -69,6 +38,41 @@
 #@   return 0
 #@ end
 
+
+#@ valid_pci_devices = [[0x10DE,0x1EB8]]
+
+#@ def valid_pci_devices_list():
+#@       return ",".join(map(lambda x:  "0x%X:0x%X" % (x[0], x[1]), valid_pci_devices))
+#@ end
+
+#@ def get_pci_devices(pci_devices_string, pci_ignore_pci_device_validation):
```

Comment:
NIT: The variable name could be either pci_ignore_device_validation or ignore_pci_device_validation.

Reply:
Done